types/index.d.ts

```diff
@@ -14,7 +14,7 @@ declare namespace inject {
 
   export type DispatchFunc = http.RequestListener
 
-  export type CallbackFunc = (err: Error, response: Response) => void
+  export type CallbackFunc = (err: Error | undefined, response: Response | undefined) => void
```

Comment:
This could be a dedicated PR

Reply:
https://github.com/fastify/light-my-request/pull/256